package.json

```diff
@@ -31,11 +31,16 @@
   },
   "devDependencies": {
     "chai": "^4.2.0",
+    "chai-http": "^4.3.0",
     "cross-env": "^7.0.2",
+    "did-io": "^0.8.3",
+    "did-method-key": "^0.7.0",
```

Comment:
If at all possible, let's use the `@digitalbazaar/` versions of those packages.

Reply:
Addressed